lib/jekyll-relative-links/generator.rb

```diff
@@ -24,9 +24,9 @@ def generate(site)
       @site    = site
       @context = context
 
-      site.pages.each do |page|
+      site.each_site_file do |page|
         next unless markdown_extension?(page.extname)
-        url_base = File.dirname(page.path)
+        url_base = File.dirname(page.relative_path)
```

Comment:
`Document#path` returns an absolute path, so switched this to `#relative_path`, which appears to work on`Page` too.